testData/solutions/win32.json

```diff
@@ -919,7 +922,8 @@
                         "SizeWE": "REG_SZ",
                         "UpArrow": "REG_SZ",
                         "Wait": "REG_SZ"
-                    }
+                    },
+                    "verifySettings": true
```

Comment:
Sorry, I misread this one - it does actually use SPI for the lifecycle action and so should be verified